src/content/en/2022/css.md

```diff
@@ -7,7 +7,7 @@ reviewers: [svgeesus, j9t]
 analysts: [rviscomi]
 editors: [rviscomi]
 translators: []
-rachelandrew_bio: Rachel Andrew is a web developer and technical writer and editor, working for Google on the Chrome team as a staff technical writer and content lead.
+rachelandrew_bio: Rachel Andrew works for Google as a technical writer, working on [web.dev](https://web.dev) and the [Chrome Developers site](https://developer.chrome.com/). She is a front and back-end web developer, author and speaker, author or co-author of 22 books including [The New CSS Layout](https://abookapart.com/products/the-new-css-layout) and a regular contributor to a number of publications both on and offline. Rachel is a Member of the CSS Working Group, and can be found posting photos of her cats on Twitter as [@rachelandrew](https://twitter.com/rachelandrew). 
```

Comment:
Slight tweak as we internationalise links with an `hreflang` to show a symbol on translations (except a few sites like Twitter that have great internatiionalised support):

```suggestion
rachelandrew_bio: Rachel Andrew works for Google as a technical writer, working on <a hreflang="en" href="https://web.dev">web.dev</a> and the <a hreflang="en" href="https://developer.chrome.com/">Chrome Developers site</a>. She is a front and back-end web developer, author and speaker, author or co-author of 22 books including <a hreflang="en" href="https://abookapart.com/products/the-new-css-layout">The New CSS Layout</a> and a regular contributor to a number of publications both on and offline. Rachel is a Member of the CSS Working Group, and can be found posting photos of her cats on Twitter as [@rachelandrew](https://twitter.com/rachelandrew). 
```